js_src/rdm.js

```diff
@@ -1,146 +1,149 @@
-            var checksum1 = checksum & 0xff;
-
-            euid_bytes.push(checksum0 | 0xaa);
-            euid_bytes.push(checksum0 | 0x55);
-            euid_bytes.push(checksum1 | 0xaa);
-            euid_bytes.push(checksum1 | 0x55);
-
-            convertor.euid = euid_bytes.map(function (i) {
-                return i.toString(16);
-            }).join(' ');
-        };
-    }])
-    .controller('EUIDController',['$scope', function (convertor) {
-        convertor.euid = '';
-        convertor.error = '';
-        convertor.uid = '';
-
-        convertor.convertToUID = function () {
-            convertor.error = '';
-            convertor.uid = '';
-
-            var tokens = convertor.euid.trim().split(' ');
-            if (tokens.length != 16) {
-                convertor.error = 'Invalid EUID, only found ' + tokens.length +
-                ' bytes';
-                return;
-            }
-
-            var data = [];
-            for (var i = 0; i < tokens.length; i++) {
-                var value = parseInt(tokens[i], 16);
-                if (isNaN(value)) {
-                    convertor.error = 'Invalid EUID: ' + tokens[i];
-                    return;
-                }
-                // parseInt may not use the entire string, so 0xfg comes back
-                // as 15.
-                if (value.toString(16) != tokens[i]) {
-                  convertor.error = 'Invalid EUID: ' + tokens[i];
-                  return;
-                }
-                data.push(value);
-            }
-
-            var manufacturer0 = data[0] & data[1];
-            var manufacturer1 = data[2] & data[3];
-            var manufacturer = (manufacturer0 << 8) + manufacturer1;
-
-            var device0 = data[4] & data[5];
-            var device1 = data[6] & data[7];
-            var device2 = data[8] & data[9];
-            var device3 = data[10] & data[11];
-
-            var device = (device0 << 24) + (device1 << 16) + (device2 << 8) + device3;
-
-            var recovered_checksum = ((data[12] & data[13]) << 8) +
-                (data[14] & data[15]);
-
-            var calculated_checksum = data.slice(0, 12).reduce(
-                function(previousValue, currentValue, index, array) {
-                  return previousValue + currentValue;
-                }
-            );
-
-            if (recovered_checksum != calculated_checksum) {
-                convertor.error = 'Checksum mismatch, was ' + recovered_checksum +
-                ', calculated to be ' + calculated_checksum;
-                return;
-            }
-
-            function zeroPad(num, places) {
-                var str = num.toString(16);
-                var zero = places - str.length + 1;
-                return Array(+(zero > 0 && zero)).join("0") + str;
-            }
-
-            convertor.uid = zeroPad(manufacturer) + ':' + zeroPad(device, 8);
-        };
-    }]);
+ .config(['$interpolateProvider', function ($interpolateProvider) {
```

Comment:
yes it's 'white' but that also disable the checks for: if, for, while, switch and catch i believe. It's kinda hard to know because the docs for jshint 2.4.4 are not easy to find because it's a old version.